ui/src/variables/actions/thunks.ts

```diff
@@ -110,20 +131,24 @@ export const hydrateVariables = (skipCache?: boolean) => async (
     skipCache,
   }).promise
 
-  vars
-    .filter(v => {
-      return vals[v.id] && v.arguments.type === 'query'
-    })
-    .forEach(v => {
-      v.arguments.values.results = vals[v.id].values
-      v.selected = vals[v.id].selected
-    })
+  const lookup = vals.reduce((prev, curr) => {
```

Comment:
this change just reflects that hydratevars returns a list of variables now instead of the variablevalues[] thing